web/src/views/DandisetLandingView/ShareDialog.vue

```diff
@@ -7,7 +7,10 @@
     max-width="500"
   >
     <template #activator="{ on }">
-      <div v-on="on">
+      <div
+        class="d-inline"
+        v-on="on"
+      >
         <v-icon
           color="primary"
         >
```

Comment:
I think setting [`cursor` to `pointer`](https://developer.mozilla.org/en-US/docs/Web/CSS/cursor) here would make it more obvious to the user that the icon is clickable

Reply:
not sure if I understand

Reply:
If you roll your mouse over this share icon, you'll see that the mouse pointer stays an arrow; if you set the CSS `cursor` value to `pointer` for the `<v-icon>` element, instead it will display as the clicking finger that appears when you roll over a hyperlink.

You can set a class on the `<v-icon>` element and then add a rule to the `<style>` section, or you can (perhaps, not entire sure) inline the style on the `<v-icon>` itself. @mvandenburgh, is there a preferred option here (including a third way I didn't mention)?

Reply:
Since vuetify makes heavy use of utility classes like `d-inline` above for styling, I usually prefer inline styles if applying the style to a single component so that you can see both the classes + styles in one glance (and generally that's how its done in the rest of the codebase). But AFAIK there's no best practice for this sort of thing.